### PR TITLE
Change for extension

### PR DESCRIPTION
**Pull request by yyongycy**
### What changes are proposed in this pull request?

Change for extension

### Why are the changes needed?

Change for extension

### Does this PR introduce any user facing changes?
NA

### CODE REVIEW

File: dora/core/common/src/main/java/alluxio/grpc/GrpcServerBuilder.java [hunk 1]
```diff
@@ -88,6 +88,19 @@ public static GrpcServerBuilder forAddress(GrpcServerAddress serverAddress,
...
+   * Create a new instance of {@link GrpcServerBuilder} with authentication support.
+   *
+   * @param serverAddress server address
+   * @param authenticationServer the authentication server to use
+   * @param conf the Alluxio configuration
+   * @return a new instance of {@link GrpcServerBuilder}
+   */
+  public static GrpcServerBuilder forAddress(GrpcServerAddress serverAddress,
+      AuthenticationServer authenticationServer, AlluxioConfiguration conf) {
+    return new GrpcServerBuilder(serverAddress, authenticationServer, conf);
```
Comment (Jackson-Wang-7) [inline dora/core/common/src/main/java/alluxio/grpc/GrpcServerBuilder.java:101]: why do not change the construct method to public?

**Review by Jackson-Wang-7 — APPROVED** [review]
mostly looks good, left a question

File: dora/core/common/src/main/java/alluxio/grpc/GrpcServerBuilder.java (hunk 1 above)
Comment (yyongycy) [reply dora/core/common/src/main/java/alluxio/grpc/GrpcServerBuilder.java:101]: this is following how the other functions works.

**Comment by yyongycy** [comment]
alluxio-bot, merge this please

**Pull request merged**